src/coreclr/scripts/superpmi.py

```diff
@@ -649,7 +649,13 @@ def __init__(self, coreclr_args):
 
         self.collection_shim_path = os.path.join(self.core_root, self.collection_shim_name)
 
-        self.jit_path = os.path.join(coreclr_args.core_root, get_jit_name(coreclr_args))
+        jit_name = get_jit_name(coreclr_args)
+        jit_name_ext = os.path.splitext(jit_name)[1]
+        jit_name_without_ext = os.path.splitext(jit_name)[0]
+        self.jit_path = os.path.join(coreclr_args.core_root, jit_name)
+        self.superpmi_jit_path = os.path.join(coreclr_args.core_root, jit_name_without_ext + "_superpmi." + jit_name_ext)
+        shutil.copyfile(self.jit_path, self.superpmi_jit_path)
```

Comment:
Can we only do the copy if we're doing crossgen2 collection (the only place we think it is needed)?

```suggestion
        jit_name = get_jit_name(coreclr_args)
        self.jit_path = os.path.join(coreclr_args.core_root, jit_name)
        if coreclr_args.crossgen2:
                jit_name_ext = os.path.splitext(jit_name)[1]
                jit_name_without_ext = os.path.splitext(jit_name)[0]
                self.superpmi_jit_path = os.path.join(coreclr_args.core_root, jit_name_without_ext + "_superpmi." + jit_name_ext)
                shutil.copyfile(self.jit_path, self.superpmi_jit_path)
        else:
                self.superpmi_jit_path = self.jit_path
```

And then, towards the end of `collect()`, remove the copy:
```
                    if self.coreclr_args.crossgen2 and not self.coreclr_args.skip_cleanup:
                        os.remove(self.superpmi_jit_path)
```

Reply:
Sure, can make this change later tonight.